### PR TITLE
Added googleadservice tracking to landing page

### PR DESCRIPTION
**Pull request by cybrox**
:boar: Just loaded the script globally for now since it's apparently bould to some pageloads events, we might want to optimize that later on. - Or just leave it if we're gonna track other buttons (like the normal header) as well.

### CODE REVIEW

File: frontend/app/controllers/index.js [hunk 1]
```diff
@@ -18,5 +18,37 @@ export default Ember.Controller.extend({
...
-  }.property()
+  }.property(),
+
+  googReportConversion: function(url) {
+    window.google_conversion_format = "3";
+    window.google_is_call = true;
+    var opt = {
+      onload_callback: function() {
+        if (typeof(url) !== 'undefined') {
+          window.location = url;
```
Comment (NuckChorris) [inline frontend/app/controllers/index.js]: Can't we do `transitionTo`?

**Comment by NuckChorris** [comment]
Maybe `ember-cli-Google-analytics` would be better?

**Comment by JoshFabian** [comment]
Are you certain they're the same thing? This embed code came from google adwords dashboard, not google analytics.

**Comment by cybrox** [comment]
I only searched for google adservice, fml.
Does `ember-cli-google-analytics` support what we need?

**Comment by NuckChorris** [comment]
Nah, is there an Adwords tracking plugin for ember-cli?

Anyways the biggest things that bug me in this are that (1) it's just kind of crammed in the rails template (2) if Google fails, God forbid, it just stops transitioning to the sign up pages and (3) it's not using `transitionTo` which causes a reload right as we want to impress the user with our snappiness

**Comment by cybrox** [comment]
(1) As I said, I just added it globally, we need to find a better way for this. (2) Is a valid point, maybe a timeout would be a workaround. And (3) Would be possible to change with a bit of hacking, though the Facebook links were never ember'd iirc.

**Comment by NuckChorris** [comment]
The Facebook links actually can't be embered I think. May I suggest we just have a function that fire-and-forgets to GA? Then we'd just in the action put the `transitionTo`

**Comment by cybrox** [comment]
That would be the only alternative we'd have if we want to drop using the callback for redirecting - which would definitely be a good idea.

**Comment by vevix** [comment]
Don't we already have Segment, don't they have support for this?

**Comment by JoshFabian** [comment]
Segment is regular google analytics but this is google adwords tracking.

**Comment by NuckChorris** [comment]
This isn't it? https://segment.com/docs/integrations/adwords/

**Comment by JoshFabian** [comment]
Oh, actually it is! @cybrox can we track button clicks with that? Just page tracking won't work otherwise we'll have 100% conversions since everyone will be hitting the landing page. Alternatively, if we fix Facebook to properly push into onboarding we could track the first page of onboarding, we just won't know how many people are FB connecting and how many are signing up with email. These are metrics I'd like to know because it could drive what we should be placing priority on in the future.

Either way, would be great to get tracking up in the next day or so. I'd like to start pushing traffic to this page.

**Comment by vevix** [comment]
This seems to have stagnated and is pretty important. I can take this over and implement segment if @cybrox is too busy.

**Pull request merged**